pkg/scheduler/internal/queue/scheduling_queue_test.go

```diff
@@ -734,7 +734,8 @@ func Test_InFlightPods(t *testing.T) {
 				for _, pod := range pods {
 					podNames = append(podNames, pod.Name)
 				}
-				if diff := cmp.Diff(test.wantActiveQPodNames, podNames); diff != "" {
+				sortOpt := cmpopts.SortSlices(func(a, b string) bool { return a < b })
```

Comment:
can we move sortOpt somewhere else so that it's declared only once.

Reply:
done